plugins/infinispan100/src/main/java/org/radargun/service/ServerStartPattern.java

```diff
@@ -0,0 +1,25 @@
+package org.radargun.service;
+
+import java.util.regex.Pattern;
+
+/**
+ * Patterns which will be used into InfinispanServerLifecycle
+ * @author Gustavo Lira &lt;glira@redhat.com&gt;
+ */
+public enum ServerStartPattern {
```

Comment:
This name is a little confusing, since it includes more than the Start Pattern. Maybe `ServerLogPattern` instead or something similar?